docs/index.md

```diff
@@ -227,3 +241,64 @@ spec:
 
 The Rules installed in the Ruler by the operator will all have those labels appended to their list of labels.  
 This effectively has the same effect as overriding each individual alert to add a new label.
+
+### MimirAlertManagerConfig
+
+
+The MimirAlertManagerConfig CRD allows the remote control of the Alert Manager Config for a specific tenant in a Mimir from Kubernetes.  
```

Comment:
The MimirAlertManagerConfig CRD allows the remote control of the Alertmanager config for a specific tenant in a Mimir **instance** from Kubernetes.

Reply:
might be clearer